maestro/README.md

```diff
@@ -126,7 +126,7 @@ This will start the workflow in a [`streamlit`]() server and you can access it f
 
 * Install dependencies: `poetry shell && poetry install`
 
-* Install dependency workaround: pip install beeai-framework
+  * If using **crewAI**: `pip install crewai`
```

Comment:
Installing the crewai causes putting the older litellm.  The crewai works with the latest litellm anyway.  How about doing this?
```
pip install crewai litellm==1.67.0.post1
```